docs/debug/3_api_debug_setup.rst

```diff
@@ -0,0 +1,85 @@
+..
+    Copyright (c) 2022-2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+
+    See LICENSE for license information.
+
+Setup
+=====
+
+Precision debug tools for the Transformer Engine use `Nvidia-DL-Framework-Inspect <https://github.com/NVIDIA/nvidia-dlfw-inspect>`_ package from NVIDIA. Here, we provide brief information on how to use it with TE.
```

Comment:
Should we link to some documentation of that package somewhere? e.g. the initialize function seems like their API and has nothing to do with TE